recipes/sphinxbase/meta.yaml

```diff
@@ -1,12 +1,12 @@
 {% set name = "sphinxbase" %}
-{% set version = "0.8" %}
+{% set version = "5prealpha" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
```

Comment:
I don't think `5prealpha` is a valid conda-forge version. Would it be possible to write some jinja2 to convert `5prealpha` to a valid version? maybe `0.5.0.prealpha`?

Reply:
Ubuntu uses `0.8+5prealpha`

Reply:
ah ok, would that be a valid version for us?

Reply:
I'm open to whatever you guys think is best in this situation.

Reply:
probably `0.8.5prealpha`